.gitlab-ci.yml

```diff
@@ -68,16 +68,20 @@ integration-tests:
   image: paritytech/ci-unified:bullseye-1.70.0
   stage: test
   variables:
-    CI: "true"
+    SPIRITNET_BLOCK_NUMBER: 6390611
+    HYDRADX_BLOCK_NUMBER: 5235787
+    POLKADOT_BLOCK_NUMBER: 21010819 
+    SPIRITNET_WASM_OVERRIDE: "../../target/debug/wbuild/spiritnet-runtime/spiritnet_runtime.compact.compressed.wasm" 
```

Comment:
No compact WASM is generated anymore, so this should refer to the regular, uncompressed, uncompacted one.